src/common/roll_renderer.js

```diff
@@ -410,13 +410,16 @@ class Beyond20RollRenderer {
             let d20_modifier = request.reliableTalent ? "min10" : "";
             if (request.silverTonge && (request.skill === "Deception" || request.skill === "Persuasion"))
                 d20_modifier = "min10";
+            if (request.indomitableMight && request.skill === "Athletics")
+                d20_modifier = request.indomitableMight ? `min${(parseInt(request.character.abilities[0][2])-parseInt(request.modifier))}` : "";
```

Comment:
You don't need to check indomitableMight here, you're already in an `if` that checks it